listenbrainz/domain/tests/test_spotify.py

```diff
@@ -51,6 +52,26 @@ def test_refresh_user_token(self, mock_update_token, mock_get_spotify_oauth, moc
         )
         mock_get_user.assert_called_with(self.spotify_user.user_id)
 
+    @requests_mock.Mocker()
+    def test_refresh_user_token_bad(self, mock_requests):
```

Comment:
both this and the other test have the same name, which means that only one of them is being run :(

Reply:
:facepalm: . Fixed now.